### PR TITLE
Replace glog with klog

### PR DESCRIPTION
**Pull request by chelseychen**
#### What type of PR is this?

/kind feature

#### Which issue(s) this PR fixes:
This PR is to replace usage of glog with klog.

#### Does this PR introduce a user-facing change?

```release-note
NONE
```

#### Additional documentation e.g., KEPs (Kubernetes Enhancement Proposals), usage docs, etc.:

```docs

```

### CODE REVIEW

**Comment by chelseychen** [comment]
cc @kwiesmueller @raywainman

File: addon-resizer/nanny/nanny_lib.go [hunk 1]
```diff
@@ -168,23 +168,21 @@ func updateResources(k8s KubernetesClient, est ResourceEstimator, now, lastChang
 	// If there's a difference, go ahead and set the new values.
 	overwriteRes, op := shouldOverwriteResources(int64(threshold), resources.Limits, resources.Requests, expResources.Limits, expResources.Requests)
 	if !overwriteRes {
-		if log.V(4) {
```
Comment (kwiesmueller) [inline addon-resizer/nanny/nanny_lib.go:171]: This change could have performance implications. Does klog have the if option too?

**Review by kwiesmueller — COMMENTED** [review]
/ok-to-test

File: addon-resizer/nanny/nanny_lib.go (hunk 1 above)
Comment (chelseychen) [reply addon-resizer/nanny/nanny_lib.go:171]: Replaced with klog and it has error `non-boolean condition in if statement` when trying to build the image

Comment (chelseychen) [reply addon-resizer/nanny/nanny_lib.go:171]: ohh looks like I should use `if klog.V(2).Enabled() `

Comment (chelseychen) [reply addon-resizer/nanny/nanny_lib.go:171]: Done.

**Review by kwiesmueller — APPROVED** [review]
/lgtm
/approve

**Pull request merged**